src/AppBundle/Command/AddUserCommand.php

```diff
@@ -36,17 +37,23 @@
  *
  * @author Javier Eguiluz <javier.eguiluz@gmail.com>
  * @author Yonel Ceruto <yonelceruto@gmail.com>
+ * @author Romain Marecat <romain.marecat@gmail.com>
  */
-class AddUserCommand extends ContainerAwareCommand
+class AddUserCommand extends Command
 {
     const MAX_ATTEMPTS = 5;
 
     /**
-     * @var ObjectManager
+     * @var ObjectManager $entityManager
```

Comment:
addig the property name does not make sense. We are documenting a property definition here